Packs/Zoom/Integrations/Zoom/README.md

```diff
@@ -63,40 +64,61 @@ In the Team Chat Subscription section under BOT endpoint URL, add:
    | Scope Type | Scope Name | 
    | --- |  --- | 
    | Team Chat |  Enable Chatbot within Zoom Team Chat Client /imchat:bot |
-   | Team Chat |  Send a team chat message to a Zoom Team Chat user or channel on behalf of a Chatbot /imchat:write:admin |
-   | Team Chat |  View and manage all users' team chat channels /chat_channel:write:admin |
-   | User |   View all user information /user:read:admin |
+   | Team Chat |  View a user's chat message /team_chat:read:user_message:admin |
+   | Team Chat |  View a user's chat sessions /team_chat:read:list_user_sessions:admin |
+   | Team Chat |  View a user's chat messages /team_chat:read:list_user_messages:admin |
+   | Team Chat |  Create a chat channel for a user /team_chat:write:user_channel:admin |
+   | Team Chat |  Add members to a chat channel /team_chat:write:members:admin |
+   | Team Chat |  Update a user's chat channel /team_chat:update:user_channel:admin |
+   | User |   View all user information /user:read:user:admin |
 ![enter image description here](../../doc_files/scope-premissions.png)
 
-3. Click **Local Test** > **Add** to test your app and authorize your Cortex XSOAR app.
+1. Click **Local Test** > **Add** to test your app and authorize your Cortex XSOAR app.
    
  ![enter image description here](../../doc_files/test-zoom-app.gif)
 
 4. **If mirroring is enabled in the integration configuration or using ZoomAsk**:
 
-    **Endpoint URL Requirements:**
-   
-      To receive webhooks, the Event notification endpoint URL that you specify for each event subscription must:   
-      * Be a publicly accessible https endpoint url that supports TLSv1.2+ with a valid certificate chain issued by a Certificate Authority (CA).
-      * Be able to accept HTTP POST requests.
-      * Be able to respond with a 200 or 204 HTTP Status Code.
-    ---
-    1. Copy the **secret token** from the "Feature" page under the "Token" section and add it to the instance configuration.
+**Endpoint URL Requirements:**
+
+&emsp; &emsp; To receive webhooks, the Event notification endpoint URL that you specify for each event subscription must:
```

Comment:
Let's add a link here to show these requirements actually come from Zoom? https://developers.zoom.us/docs/api/webhooks/#webhook-endpoint-requirements